### ISSUE TITLE
Standard Library documentation + examples

### ISSUE THREAD

**Issue by NoahTheDuke** [OCamlverse/ocamlverse.github.io#201]
Coming from Clojure, which has clojuredocs.org, I'm interested in a space for user-provided usage examples of the functions of the standard library. Is that kind of feature appropriate for ocamlverse? If so, I'm willing to help build it.

**Comment by bluddy** [comment]
Yeah, definitely! Figure out where it should be on the hierarchy and go for it - we have no problem merging partial content if it looks good.
